src/java/Logica/Entidades/Valoracion.java

```diff
@@ -0,0 +1,76 @@
+package Logica.Entidades;
+
+import java.io.Serializable;
+import javax.persistence.Basic;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Valoracion implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Integer id;
+
+    @Basic
+    private Integer idPaper; // ID del paper asociado
```

Comment:
@patricioperrone sucede lo mismos que para Lector. Utilizar la entity Paper con la anotacion @OneToOne.
Esto seria: 
`@OneToOne
private Paper paper`
Revisar si es necesario agregar alguna configuracion extra en el OneToOne https://www.geeksforgeeks.org/jpa-one-to-one-mapping/
esto ultimo le podes preguntar al chatgpt